lib/start_game.rb

```diff
@@ -1,21 +1,32 @@
 class StartGame
   def initialize(cpu_board, player_board)
-    @cpu_board = cpu_board
-    @player_board = player_board
+    setup(cpu_board, player_board)
   end
 
   def start
-  #  intro
-  #  user_ready?
-  #  comp_ship_placement
-  #  cpu_board.board_render(true)
-    player_ship_placement
-    player_board.board_render(true)
+    intro
+    user_ready?
   end
 
   private
 
-  attr_reader :cpu_board, :player_board
+  attr_reader :cpu_board, :player_board, :player_cruiser, :player_submarine, :cpu_cruiser, :cpu_submarine
+
+  def setup(cpu_board, player_board)
+    @cpu_board = cpu_board
+    @player_board = player_board
+    @player_cruiser = Ship.new('Cruiser', 3)
+    @player_submarine = Ship.new('Submarine', 2)
+    @cpu_cruiser = Ship.new('Cruiser', 3)
+    @cpu_submarine = Ship.new('Submarine', 2)
+  end
```

Comment:
Added setup and reset method to for object reinitialization which allows the game to restart